soh/soh/OTRGlobals.cpp

```diff
@@ -736,8 +736,9 @@ std::string GetName(const char* path) {
 }
 
 extern "C" const char* ResourceMgr_GetName(const char* path) {
-    auto s = new std::string(GetName(path));
-    const char* name = s->c_str();
+    std::string s = GetName(path);
+    char* name = new char[s.size() + 1];
```

Comment:
There is memory allocated to the heap that is never freed. @briaguya-ai and I are working on a patch that will actually fix the memory leak. Thankyou!